README.md

```diff
@@ -1,62 +1,59 @@
 # Navigate website
+
 Website developed using [React](https://reactjs.org/) + Typescript.
 
 This tool will be used to configure the telescope and its subsystems to point and track a celestial object during an observation. Navigate is a web application with a single centralized server that communicates with the different real-time control systems of the telescope. It also will communicate with other apps in the Gemini Program Platform (GPP) ecosystem, such as Observe and Chronicle.
 
 ## Launch on local development
-Make sure you have installed [NodeJS](https://nodejs.org/en/) in your machine.
-
-### Using yarn
-- Install dependencies
-  ```bash
-  yarn
-  ```
 
-- Run the web app
-  ```bash
-  yarn dev
-  ```
+Make sure you have installed [NodeJS](https://nodejs.org/en/) and [ni](https://github.com/antfu/ni) in your machine.
```

Comment:
`ni` is useful but the package manager used in `pnpm` so that's probably what should be in the readme

Reply:
We can also promote the use of ni, it's great. If we switch package managers, we don't have to change the README.